### PR TITLE
[Tests-Only] remove outdated issues from expected failures file

### PR DESCRIPTION
**Pull request by dpakach**
## Description

[Tests-Only] remove outdated issues from expected failures file #2158

## Related Issue

-  Related https://github.com/owncloud/ocis/issues/2078

## How Has This Been Tested?

:robot:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/expected-failures-API-on-OCIS-storage.md [hunk 1]
```diff
@@ -73,25 +72,33 @@ Basic file management like up and download, move, copy, properties, trash, versi
...
 -   [apiWebdavUploadTUS/optionsRequest.feature:46](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiWebdavUploadTUS/optionsRequest.feature#L46)
 
+#### [renaming to banned name works](https://github.com/owncloud/ocis/issues/1295)
+-   [apiWebdavMove1/moveFolder.feature:21](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiWebdavMove1/moveFolder.feature#L21)
+-   [apiWebdavMove1/moveFolder.feature:22](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiWebdavMove1/moveFolder.feature#L22)
+-   [apiWebdavMove1/moveFolder.feature:34](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiWebdavMove1/moveFolder.feature#L34)
+-   [apiWebdavMove1/moveFolder.feature:35](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiWebdavMove1/moveFolder.feature#L35)
+-   [apiWebdavMove1/moveFolder.feature:47](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiWebdavMove1/moveFolder.feature#L47)
+-   [apiWebdavMove1/moveFolder.feature:48](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiWebdavMove1/moveFolder.feature#L48)
+
```
Comment (jasson99) [inline tests/acceptance/expected-failures-API-on-OCIS-storage.md:82]: Why did we move it here?

File: tests/acceptance/expected-failures-API-on-OCIS-storage.md [hunk 2]
```diff
@@ -961,21 +911,17 @@ Scenario Outline: delete a folder when there is a default folder for received sh
 -   [apiWebdavOperations/deleteFolder.feature:92](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiWebdavOperations/deleteFolder.feature#L92)
 
 #### [sharing with group not available](https://github.com/owncloud/product/issues/293)
```
Comment (jasson99) [inline tests/acceptance/expected-failures-API-on-OCIS-storage.md:913]: This is a closed issue. Lets not add here.

File: tests/acceptance/expected-failures-API-on-OWNCLOUD-storage.md [hunk 3]
```diff
@@ -31,11 +30,13 @@ The following scenarios fail on OWNCLOUD storage but not on OCIS storage:
...
 -   [apiTrashbin/trashbinFilesFolders.feature:279](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiTrashbin/trashbinFilesFolders.feature#L279)
 
-#### [href in trashbin PROPFIND response is wrong](https://github.com/owncloud/ocis/issues/1120)
+### [[Trashbin] restoring a file in existing location gives incorrect status code](https://github.com/owncloud/ocis/issues/2139)
 -   [apiTrashbin/trashbinRestore.feature:116](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiTrashbin/trashbinRestore.feature#L116)
 -   [apiTrashbin/trashbinRestore.feature:117](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiTrashbin/trashbinRestore.feature#L117)
 -   [apiTrashbin/trashbinRestore.feature:118](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiTrashbin/trashbinRestore.feature#L118)
 -   [apiTrashbin/trashbinRestore.feature:119](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiTrashbin/trashbinRestore.feature#L119)
+
+#### [href in trashbin PROPFIND response is wrong](https://github.com/owncloud/ocis/issues/1120)
```
Comment (jasson99) [inline tests/acceptance/expected-failures-API-on-OWNCLOUD-storage.md:39]: this issue is closed. Lets not add here

**Review by jasson99 — CHANGES_REQUESTED** [review]

**Review by kiranparajuli589 — COMMENTED** [review]
lgtm 👍🏼

**Review by Talank — COMMENTED** [review]
LGTM

**Review by jasson99 — APPROVED** [review]

**Review by phil-davis — APPROVED** [review]

**Pull request merged**